projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-theme.scss

```diff
@@ -282,8 +285,8 @@
         igx-list {
             flex-grow: 1;
             overflow: hidden;
-            margin-left: -#{rem(16px)};
-            margin-right: -#{rem(16px)};
+            margin-inline-start: -#{rem(16px)};
```

Comment:
you can write just margin-inline: -#{rem(16px)}; and it'll cover both properties